File: app/src/main/java/net/tlalka/android/fiszki/model/dao/ClusterDao.java
package net.tlalka.android.fiszki.model.dao;

import java.sql.SQLException;

import com.j256.ormlite.support.ConnectionSource;
import net.tlalka.android.fiszki.model.entities.Cluster;

public class ClusterDao extends AbstractDao<Cluster, Long> {

    public ClusterDao(ConnectionSource connectionSource) throws SQLException {
        super(connectionSource, Cluster.class);
    }
}

File: app/src/main/java/net/tlalka/android/fiszki/model/dao/ClusterDao.kt
package net.tlalka.android.fiszki.model.dao

import com.j256.ormlite.support.ConnectionSource
import net.tlalka.android.fiszki.model.entities.Cluster
import java.sql.SQLException

class ClusterDao @Throws(SQLException::class)
constructor(connectionSource: ConnectionSource) : AbstractDao<Cluster, Long>(connectionSource, Cluster::class.java)
